Main.java

```diff
@@ -0,0 +1,83 @@
+package com.company;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+
+        int[] arr1 = new int[]{1, 2, 3, 4, 5};
+        int[] arr2 = new int[]{2, 3, 4, 6, 10};
+        int[] arr3 = mergeArray(arr1, arr2);
+
+        Arrays.stream(arr3).forEach((i) -> System.out.print(arr3[i - 1] + " "));

```

Comment:
just System.out.print(i) i is element of your array. This will fail if you add 13 to your arr1